minIO/minIO.py

```diff
@@ -1,14 +1,17 @@
 from minio import Minio
 import io
-from db.db import ImageDao
 
 class ImageMinioAccess:
 
-    def __init__(self): 
-        self.client = Minio(endpoint = 'play.min.io',
-        access_key = 'Q3AM3UQ867SPQQA43P2F',
-        secret_key = 'zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG')
+    def __init__(self, endpoint, access_key, secret_key): 
```

Comment:
```suggestion
    def __init__(self, endpoint, access_key, secret_key, bucket_name):
```

И так далее с возможностью сменить это имя